app/components/data/index.js

```diff
@@ -373,97 +374,16 @@ export const findRecommendedRoommates = async({
 }) => {
     const {
         maxRecommendedRoommates,
-        minResponse,
-        maxResponse
     } = surveyContants;
 
-    // First get all the roommate survey responses that are looking for the same city as our user
-    let roommateSurveyResponses = [];
-
-    try {
-        roommateSurveyResponses = await roommateSurveysCollection.aggregate([
-            {
-                $match: {
-                    city: userSurveyResponse.city,
-                    userId: {
-                        $ne: userSurveyResponse.userId
-                    }
-                }
-            },
-            {
-                $lookup: {
-                    from: 'users',
-                    localField: 'userId',
-                    foreignField: '_id',
-                    as: 'users'
-                }
-            }
-        ]).toArray();
-
-        // NOTE: Cheesey map to take 1 user out of the users because we don't have time to deal with the db upgrade to 3.4
-        roommateSurveyResponses = roommateSurveyResponses
-            .map(({
-                users,
-                userId,
-                ...rest
-            }) => ({
-                user: users[0],
-                ...rest
-            }))
-            .filter(x => !x.user.isLandlord);
-
-    } catch (e) {
-        throw new RethrownError(e, 'Error getting roommate survey responses');
-    }
-
-    const {
-        userId: submittedUserId,
-        createdAt,
-        ...submittedUserQuestionResponses
-    } = userSurveyResponse;
-
-    // Compute the distance from our current user to each survey response.
-
-    const roommateDistances = roommateSurveyResponses.map((response) => {
-        const {
-            _id,
-            city,
-            user,
-            userId,
-            createdAt,
-            ...questionResponses
-        } = response;
-        // Compute the Euclidean distance between the current response and our user
-        const squaredDistance = Object.keys(questionResponses)
-            .reduce((accumulator, responseKey) => {
-                const componentDistance = Math.pow(submittedUserQuestionResponses[responseKey] - questionResponses[responseKey], 2);
-
-                return accumulator + componentDistance;
-            }, 0);
-
-        const distance = Math.sqrt(squaredDistance);
-        const maxDistance = Object.keys(questionResponses).length * Math.pow(maxResponse - minResponse, 2);
-        const percentMatch = 100 - distance / maxDistance;
-
-        return {
-            user: {
-                ...user,
-                percentMatch
-            },
-            distance
-        };
+    const result = await axios.get(`${process.env.JAVA_FIND_ROOMMATE_URL}`,{
+        'headers':{
```

Comment:
These should be query parameters, not in the header

Reply:
So essentially just put this stuff in a `params` object instead of a `headers` object. And then change the java app to look for query parameters instead of in the header.